hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,220 @@
+            name: deployment-validation-operator
+            namespace: openshift-acm-policies
+          spec:
+            disabled: false
+            remediationAction: enforce
+            policy-templates:
+              - objectDefinition:
+                  apiVersion: policy.open-cluster-management.io/v1
+                  kind: ConfigurationPolicy
+                  metadata:
+                    name: deployment-validation-operator
+                  spec:
+                    evaluationInterval:
+                      compliant: 2h
+                      noncompliant: 45s
+                    object-templates:
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: v1
+                          kind: Namespace
+                          metadata:
+                            name: openshift-deployment-validation-operator
+                            labels:
+                              openshift.io/cluster-monitoring: "true"
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: operators.coreos.com/v1alpha1
+                          kind: CatalogSource
+                          metadata:
+                            name: deployment-validation-operator-catalog
+                            namespace: openshift-deployment-validation-operator
+                          spec:
+                            sourceType: grpc
+                            image: ${REGISTRY_IMG}@${IMAGE_DIGEST}
+                            displayName: Deployment Validation Operator
+                            publisher: Red Hat
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: operators.coreos.com/v1
+                          kind: OperatorGroup
+                          metadata:
+                            name: deployment-validation-operator-og
+                            namespace: openshift-deployment-validation-operator
+                            annotations:
+                              olm.operatorframework.io/exclude-global-namespace-resolution: "true"
+                          spec:
+                            upgradeStrategy: TechPreviewUnsafeFailForward
+                            targetNamespaces:
+                              - openshift-deployment-validation-operator
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: operators.coreos.com/v1alpha1
+                          kind: Subscription
+                          metadata:
+                            name: deployment-validation-operator
+                            namespace: openshift-deployment-validation-operator
+                          spec:
+                            channel: ${CHANNEL}
+                            config:
```

Comment:
Affinity of the operator to infra nodes has to be removed.
Here the code to add under `config`:
```
config:
  affinity:
    nodeAffinity: {}
```

Reply:
I am not sure I understand here. So the affinity default value is not an empty object/type `{}`?

Reply:
ohh so this is to override the https://github.com/app-sre/deployment-validation-operator/blob/master/deploy/openshift/operator.yaml#L30 right? Then it makes sense.....probably :)

Reply:
Yes this overrides that config with an empty (`{}`) config.